src/main/kotlin/com/openlattice/chronicle/tud/TudResponse.kt

```diff
@@ -0,0 +1,16 @@
+package com.openlattice.chronicle.tud
+
+import java.time.OffsetDateTime
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+/**
+ * @author alfoncenzioka &lt;alfonce@openlattice.com&gt;
+ *
+ */
+
+data class TudResponse(
+        @JsonProperty("questionCode") val questionCode: Set<String>,
+        @JsonProperty("response") val response: Set<Any>,
+        @JsonProperty("startDateTime") val startDateTime: Set<OffsetDateTime>?,
+        @JsonProperty("endDateTime") val endDateTime: Set<OffsetDateTime>?
```

Comment:
If `startDateTime` and `endDateTime` are not passed, what would you expect the value of these objects to be? The empty set? I've seen other `data class` files initialize to null like `? = null`.

Additionally, the naming is a bit confusing. Since these are sets, should the name be plural?

Reply:
I set the default value to be the empty set as `startDateTime` and `endDateTime` are only absent when the survey doesn't ask for datetime info. If it was the case that the survey requests datetime data but the user gave no input, then I would've said `null` should've been the default value. Let me know if you feel otherwise.

`startDateTime` and `endDateTime` are singular because the Set wrapping the values only ever contains one value. I believe the singular value is wrapped in a Set because of how the front end generates the JSON as we discussed in Slack. Adding that context here for documentation purposes.